packages/cloudflare/README.md

```diff
@@ -186,9 +188,9 @@ export default defineConfig({
 
 ### `runtime`
 
-`runtime: "off" | "local"`
+`runtime: "off" | "local" | { mode: "off" | "local", persistTo: string }`
 
-default `"off"`
+default `{ mode: 'off', persistTo: '' }`
```

Comment:
- We should document what's `persistTo` and explain to the user how it works.
- Also, it seems that `"off"` is basically `{ mode: "off", persistTo: '' }`, and we deprecated it. The documentation should mention it.
- `remote` is missing.

Reply:
Agreed.

We purposely chose to not include `remote` since it is not yet fully implemented.